vm/class.go

```diff
@@ -1144,24 +1180,35 @@ func builtinClassCommonInstanceMethods() []*BuiltinMethodObject {
 			// @return [Boolean] Result of loading module
 			Name: "require",
 			Fn: func(receiver Object, sourceLine int) builtinMethodBody {
-				return func(t *thread, args []Object, blockFrame *normalCallFrame) Object {
+				return func(t *Thread, args []Object, blockFrame *normalCallFrame) Object {
 					if len(args) != 1 {
-						return t.vm.initErrorObject(errors.ArgumentError, sourceLine, "Expect 1 argument. got: %d", len(args))
+						return t.vm.InitErrorObject(errors.ArgumentError, sourceLine, "Expect 1 argument. got: %d", len(args))
 					}
 					switch args[0].(type) {
 					case *StringObject:
 						libName := args[0].(*StringObject).value
 						initFunc, ok := standardLibraries[libName]
 
 						if !ok {
-							return t.vm.initErrorObject(errors.InternalError, sourceLine, "Can't require \"%s\"", libName)
+							externalClassLock.Lock()
```

Comment:
is this for preventing concurrent requiring?

Reply:
Yes. Requiring happens so infrequently, I think havinga  lock is fine for now.